examples/config/config

```diff
@@ -516,7 +517,7 @@ set ssl_policy fail
 
 # === Post-load misc commands ================================================
 spawn_sync_exec @scripts_dir/load_cookies.sh
-spawn_sync_exec @scripts_dir/load_cookies.sh @data_home/session-cookies.txt
+spawn_sync_exec @scripts_dir/load_cookies.sh @(echo "${UZBL_SESSION_COOKIE_FILE:-@data_home/session-cookies.txt}")@
```

Comment:
Here, the UZBL_SESSION_COOKIE_FILE environment variable is used.

It it is not set, the set command falls back to the same default value than used in the cookies plugins and the scripts.